### ISSUE TITLE
Layout/EmptyLineAfterGuardClause fails with undefined method `blank?' for nil:NilClass

### ISSUE THREAD

**Issue by bayandin** [rubocop-hq/rubocop#6263]
Rubocop Layout/EmptyLineAfterGuardClause fais while inspecting some particular code with `undefined method `blank?' for nil:NilClass`

--------

## Expected behavior

I suppose, the error shouldn't occur

## Actual behavior

```
For /Users/bayandin/work/examples: configuration from /Users/bayandin/work/examples/.rubocop.yml
Inheriting configuration from /Users/bayandin/work/examples/.rubocop_todo.yml
Default configuration from /Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/config/default.yml
Inheriting configuration from /Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/config/enabled.yml
Inheriting configuration from /Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/config/disabled.yml
Inspecting 1 file
Scanning /Users/bayandin/work/examples/rubocop_example.rb
An error occurred while Layout/EmptyLineAfterGuardClause cop was inspecting /Users/bayandin/work/examples/rubocop_example.rb:1:0.
undefined method `blank?' for nil:NilClass
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/layout/empty_line_after_guard_clause.rb:84:in `next_line_empty?'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/layout/empty_line_after_guard_clause.rb:53:in `on_if'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/commissioner.rb:58:in `block (2 levels) in trigger_responding_cops'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/commissioner.rb:106:in `with_cop_error_handling'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/commissioner.rb:57:in `block in trigger_responding_cops'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/commissioner.rb:56:in `each'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/commissioner.rb:56:in `trigger_responding_cops'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/ast/traversal.rb:50:in `block in on_begin'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/ast/traversal.rb:50:in `each'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/ast/traversal.rb:50:in `on_begin'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/commissioner.rb:35:in `block (2 levels) in <class:Commissioner>'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/ast/traversal.rb:13:in `walk'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/commissioner.rb:46:in `investigate'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/team.rb:115:in `investigate'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/team.rb:103:in `offenses'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cop/team.rb:44:in `inspect_file'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:269:in `inspect_file'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:216:in `block in do_inspection_loop'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:248:in `block in iterate_until_no_changes'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:241:in `loop'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:241:in `iterate_until_no_changes'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:212:in `do_inspection_loop'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:115:in `block in file_offenses'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:133:in `file_offense_cache'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:113:in `file_offenses'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:101:in `process_file'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:78:in `block in each_inspected_file'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:75:in `each'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:75:in `reduce'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:75:in `each_inspected_file'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:67:in `inspect_files'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/runner.rb:39:in `run'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cli.rb:157:in `execute_runner'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cli.rb:85:in `execute_runners'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/lib/rubocop/cli.rb:41:in `run'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/exe/rubocop:13:in `block in <top (required)>'
/Users/bayandin/.rvm/rubies/ruby-2.5.1/lib/ruby/2.5.0/benchmark.rb:308:in `realtime'
/Users/bayandin/.rvm/gems/ruby-2.5.1/gems/rubocop-0.59.0/exe/rubocop:12:in `<top (required)>'
/Users/bayandin/.rvm/gems/ruby-2.5.1/bin/rubocop:23:in `load'
/Users/bayandin/.rvm/gems/ruby-2.5.1/bin/rubocop:23:in `<main>'
/Users/bayandin/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `eval'
/Users/bayandin/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `<main>'
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Layout/EmptyLineAfterGuardClause cop was inspecting /Users/bayandin/work/examples/rubocop_example.rb:1:0.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop-hq/rubocop/issues

Mention the following information in the issue report:
0.59.0 (using Parser 2.5.1.2, running on ruby 2.5.1 x86_64-darwin17)
Finished in 0.1923269999679178 seconds
```

## Steps to reproduce the problem

```
bundle exec rubocop --debug --only Layout/EmptyLineAfterGuardClause rubocop_example.rb
```

```ruby
raise(<<~FAIL) unless true
  #{1+1}
FAIL
1
```

## RuboCop version

```
$ bundle exec rubocop -V
0.59.0 (using Parser 2.5.1.2, running on ruby 2.5.1 x86_64-darwin17)
```

**Comment by koic** [comment]
Thanks for the feedback! I opened a PR #6273.
